test/unit/lib/configuration/variables/sources/instance-dependent/get-sls.test.js

```diff
@@ -35,22 +41,42 @@ describe('test/unit/lib/configuration/variables/sources/instance-dependent/get-s
       options: {},
     });
     serverlessInstance.init();
+    const options = {};
+    if (typeof cliOptionStage === 'string') {
+      options.stage = cliOptionStage;
+    }
     await resolve({
       servicePath: process.cwd(),
       configuration,
       variablesMeta,
       sources: { self: selfSource, sls: getSlsSource(serverlessInstance) },
-      options: {},
+      options,
       fulfilledSources: new Set(['self', 'sls']),
     });
-  });
+  };
+
+  beforeEach(async () => await initializeServerless());
```

Comment:
This is invoked prior every test, even one which invokes `initializeServerless()` on it's own. So that's not clean.

Let's remove this, and simply invoke `initializeServerless` in context of `it`'s